### PR TITLE
Fix timeout

### PR DESCRIPTION
**Pull request by imobachgs**

### CODE REVIEW

File: package/aytests-tests.spec [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 
 Name:           aytests-tests
-Version:        1.0.35
+Version:        1.0.37
```
Comment (schubi2) [inline package/aytests-tests.spec:20]: Hm, seems that I have forgotten to increase the number :-)

**Comment by schubi2** [comment]
LGTM

File: package/aytests-tests.spec (hunk 1 above)
Comment (imobachgs) [reply package/aytests-tests.spec:20]: :)

**Comment by imobachgs** [comment]
Thanks!

**Pull request merged**